python/ray/cluster_utils.py

```diff
@@ -316,3 +317,22 @@ def shutdown(self):
             self.remove_node(self.head_node)
         # need to reset internal kv since gcs is down
         ray.experimental.internal_kv._internal_kv_reset()
+
+
+def get_job_submission_client_cluster_info(
+        address: str, create_cluster_if_needed: bool) -> ClusterInfo:
+    """Get address, cookies, and metadata used for JobSubmissionClient.
+
+        Args:
+            address (str): Address without the module prefix that is passed
+                to JobSubmissionClient.
+            create_cluster_if_needed (bool): Indicates whether the cluster
+                of the address returned needs to be running. Ray doesn't
+                start a cluster before interacting with jobs, but other
+                implementations may do so.
+
+        Returns:
+            ClusterInfo object consisting of address, cookies, and metadata
+            for JobSubmissionClient to use.
+        """
+    return ClusterInfo(address="http://" + address, cookies=None, metadata={})
```

Comment:
cookies being none but metadata being {} is pretty inconsistent here -- `None`for both and default metadata in the constructor?

Reply:
yes, makes sense - updated